Developement.ipynb

```diff
@@ -0,0 +1,487 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'' was not found in history, as a file, url, nor in the user namespace.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# name: tabipy.py\n",
+    "# type: text/x-python\n",
+    "# size: 9219 bytes \n",
+    "# ----  \n",
+    "import re\n",
+    "import sys\n",
+    "PY3 = sys.version_info[0] >= 3\n",
+    "\n",
+    "try:\n",
+    "    from itertools import zip_longest  # Python 3\n",
+    "except ImportError:\n",
+    "    from itertools import izip_longest as zip_longest  # Python 2\n",
+    "from collections import Mapping\n",
+    "\n",
+    "class TableCell(object):\n",
+    "    bg_colour = None\n",
+    "    _latex_escape_table = {'&': r'\\&',\n",
+    "                           '\\\\': r'{\\textbackslash}',\n",
+    "                           '~': r'{\\textasciitilde}',\n",
+    "                           '$': '\\$',\n",
+    "                           '\\r\\n': r'{\\linebreak}',\n",
+    "                           '\\n': r'{\\linebreak}',\n",
```

Comment:
Did you mean to commit this file? I'd rather not have a copy of the source code in a notebook where it will get out of date.